tests/skimage/measure/test_fit.py

```diff
@@ -853,7 +853,7 @@ def test_ransac_is_model_valid():
     def is_model_valid(model, data):
         return False
 
-    with expected_warnings(["No inliers found"]):
+    with pytest.warns(UserWarning, match="No inliers found"):
```

Comment:
Your warning catch mechanism was triggering some NumPy warning, that crashed the test suite.

Reply:
I don't mind this change at all. Though, there are slight differences between these functions. 

I've been meaning to remove the `expected_warnings` at some point, but investing that time has never seemed reasonable yet. :sweat_smile: